apps/web/utils/actions/group.ts

```
@@ -13,216 +13,227 @@ import { findNewsletters } from "@/utils/ai/group/find-newsletters";
+
+export const deleteGroupItemAction = withActionInstrumentation(
+  "deleteGroupItem",
+  async (id: string) => {
+    const session = await auth();
+    if (!session?.user.id) return { error: "Not logged in" };
+
+    await prisma.groupItem.delete({
+      where: { id, group: { userId: session.user.id } },
+    });
+
+    revalidatePath(`/automation`);
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)